api/users/serializers.py

```diff
@@ -6,13 +8,15 @@
 
 from organisations.models import Organisation
 from organisations.serializers import UserOrganisationSerializer
-
-from .models import (
+from users.models import (
     FFAdminUser,
     UserPermissionGroup,
     UserPermissionGroupMembership,
 )
 
+if TYPE_CHECKING:
+    DjoserUserSerializer = DjoserUserSerializer[FFAdminUser]
```

Comment:
I'm intrigued as to why this has been added? I assume mypy threw some error without it?

Reply:
This was added by @khvn26 but I assume it was a mypy issue